src/OpenTelemetry.Instrumentation.StackExchangeRedis/TracerProviderBuilderExtensions.cs

```diff
@@ -54,12 +89,12 @@ public static class TracerProviderBuilderExtensions
     /// <param name="configure">Callback to configure options.</param>
     /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
     public static TracerProviderBuilder AddRedisInstrumentation(
-        this TracerProviderBuilder builder,
-        Action<StackExchangeRedisInstrumentationOptions> configure)
+                this TracerProviderBuilder builder,
+                Action<StackExchangeRedisInstrumentationOptions> configure)
```

Comment:
```suggestion
        this TracerProviderBuilder builder,
        Action<StackExchangeRedisInstrumentationOptions> configure)
```

Just reverting whitespace change.